.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,6 @@ I have:
 
 - [ ] Made sure my feature is working on the relevant real / virtual devices (native and web).
 - [ ] Written **unit tests** native (and web when implementation is different) for my feature.
-- [ ] Added a **screenshot** for UI tickets.
```

Comment:
pas forcément si évident. Peut-on plutôt modifier le check : 

Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change

Et ajouter à la section Screenshots un avertissement supplémentaire.

## Screenshots: delete if no UI change

ça permet de voir qu'un tableau vide est une PR non valide. Soit on remplit, soit on supprime.